CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 0.7.3dev
 * [Doc] Tutorial on querying excel files with pandas and jupysql ([#423](https://github.com/ploomber/jupysql/pull/423))
 * [Fix] Fix `--alias` when passing an existing engine
+* [Fix] Fix displaylimit default value to 60 (#462)
```

Comment:
Please add more details as to `displaylimit` of what

Reply:
Change to: `Fix the default value of %config SqlMagic.displaylimit to 10 (#462)`